test/io/input/strings.csv

```diff
@@ -0,0 +1,2 @@
+"mary had ";"a";" little lamb"
+"hello";" from";" row";" #2"
```

Comment:
Hmm, well you are right about that. But I don't think that will matter, since the reader accepts both \n or \0 as line "terminators".